frame/support/test/tests/pallet_ui/storage_result_query_wrong_generic_kind.stderr

```diff
@@ -0,0 +1,5 @@
+error: Invalid pallet::storage, unexpected generic argument kind, expected a type path to a `PalletError` enum variant, found `'static`
```

Comment:
Worth documenting everywhere you document it that you can only use the pallet error in here.